src/compas_rhino/geometry/curves/__init__.py

```diff
@@ -0,0 +1,28 @@
+from .nurbs import RhinoNurbsCurve
+
+try:
+    from compas.geometry import NurbsCurve
```

Comment:
Why is this in a try.. except block? 
What is the condition that would make it not importable?

Reply:
this can go away indeed. was necessary at some point in between two versions...